.github/workflows/release.yml

```diff
@@ -19,8 +19,55 @@ on:
       - 'v*'
 
 jobs:
+  bdd-test:
```

Comment:
do we need to run it at the release time also? we have already run this at the build time?

Reply:
Yes not required